Apps/Sandcastle/gallery/CZML Polygon - Intervals, Availability.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
Try replacing `dynamicPolygon` with simple geometry, e.g., a triangle on the surface with just three points, and we'll see if there is any issue.